static/js/components/ProgramFilter.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import {
+  AnonymousAccessor,
+  TermQuery,
+  SearchkitComponent,
+} from 'searchkit';
+import _ from 'lodash';
+
+import type { ProgramEnrollment } from '../flow/enrollmentTypes';
+
+export default class ProgramFilter extends SearchkitComponent {
+  props: {
+    currentProgramEnrollment: ProgramEnrollment,
+  };
+
+  _accessor = new AnonymousAccessor(query => {
+    const { currentProgramEnrollment } = this.props;
+    if (currentProgramEnrollment === null) {
+      return query;
+    }
+    return query.addFilter("program_filter", TermQuery("program.id", currentProgramEnrollment.id));
+  });
+
+
+  defineAccessor() {
+    return this._accessor;
+  }
```

Comment:
`_accessor` is our field so we only create the `AnonymousAccessor` once. `defineAccessor` is a `SearchkitComponent` method though, it's being used here to add an accessor to searchkit in a way where we can tie it into the React lifecycle and also keep it separated from the other searchy bits